### PR TITLE
Create fluent-terminal.json

### PR DESCRIPTION
**Pull request by klaidliadon**

### CODE REVIEW

File: fluent-terminal.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "homepage": "https://github.com/felixse/FluentTerminal",
+    "description": "A Terminal Emulator based on UWP and web technologies.",
+    "license": {
```
Comment (Ash258) [inline fluent-terminal.json]:
```suggestion
    "license": "GPL-2.0-only",
```

File: fluent-terminal.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "pre_install": [
+        "$thumb='8A5DE5B20B663BD6A99B44C5BA278289521F419F';",
+        "$store='TrustedPeople';",
+        "if (Get-ChildItem -Path Cert:\\LocalMachine\\$store | Where-Object {$_.Thumbprint -eq $thumb}) { return }",
+        "sudo certutil -addstore $store .\\FluentTerminal.Package_0.6.0.0_x86_x64.cer; gpupdate \/force;",
+        "sudo reg add 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\AppModelUnlock' \/t REG_DWORD \/f \/v 'AllowAllTrustedApps' \/d '1';"
+    ],
+    "installer": {
+        "script": ".\\Install.ps1 -ForceContextMenu -Force"
+    }
```
Comment (Ash258) [inline fluent-terminal.json]:
```suggestion
    },
    "checkver": "github",
    "autoupdate": {
        "url": "https://github.com/felixse/FluentTerminal/releases/download/$version/FluentTerminal.Package_$version_Test.zip"
    }
```

File: fluent-terminal.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "hash": "942c5d55a4183942a00b7cf8c3309dd1092402f5c97a479e7d6b4c0309167ae3",
+    "pre_install": [
+        "$thumb='8A5DE5B20B663BD6A99B44C5BA278289521F419F';",
+        "$store='TrustedPeople';",
+        "if (Get-ChildItem -Path Cert:\\LocalMachine\\$store | Where-Object {$_.Thumbprint -eq $thumb}) { return }",
+        "sudo certutil -addstore $store .\\FluentTerminal.Package_0.6.0.0_x86_x64.cer; gpupdate \/force;",
+        "sudo reg add 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\AppModelUnlock' \/t REG_DWORD \/f \/v 'AllowAllTrustedApps' \/d '1';"
+    ],
+    "installer": {
+        "script": ".\\Install.ps1 -ForceContextMenu -Force"
```
Comment (Ash258) [inline fluent-terminal.json]:
```suggestion
        "script": "& \"$dir\\Install.ps1\" -ForceContextMenu -Force"
```

File: fluent-terminal.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        "url": "https://raw.githubusercontent.com/felixse/FluentTerminal/master/LICENSE"
+    },
+    "version": "0.6.0", 
+    "url": "https://github.com/felixse/FluentTerminal/releases/download/0.6.0.0/FluentTerminal.Package_0.6.0.0_Test.zip",
+    "hash": "942c5d55a4183942a00b7cf8c3309dd1092402f5c97a479e7d6b4c0309167ae3",
+    "pre_install": [
+        "$thumb='8A5DE5B20B663BD6A99B44C5BA278289521F419F';",
+        "$store='TrustedPeople';",
+        "if (Get-ChildItem -Path Cert:\\LocalMachine\\$store | Where-Object {$_.Thumbprint -eq $thumb}) { return }",
+        "sudo certutil -addstore $store .\\FluentTerminal.Package_0.6.0.0_x86_x64.cer; gpupdate \/force;",
```
Comment (Ash258) [inline fluent-terminal.json]: Static version and no $dir

File: fluent-terminal.json (hunk 3 above)
Comment (linsui) [reply fluent-terminal.json]: Why `script` instead of `file` and `args`?

File: fluent-terminal.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
+    "homepage": "https://github.com/felixse/FluentTerminal",
+    "description": "A Terminal Emulator based on UWP and web technologies.",
+    "license": "GPL-2.0-only",
```
Comment (linsui) [inline fluent-terminal.json]:
```suggestion
    "license": "GPL-3.0-only",
```
Remove L5-L7

File: fluent-terminal.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        "identifier": "GNU General Public License v3.0",
+        "url": "https://raw.githubusercontent.com/felixse/FluentTerminal/master/LICENSE"
+    },
+    "version": "0.6.0", 
+    "url": "https://github.com/felixse/FluentTerminal/releases/download/0.6.0.0/FluentTerminal.Package_0.6.0.0_Test.zip",
+    "hash": "942c5d55a4183942a00b7cf8c3309dd1092402f5c97a479e7d6b4c0309167ae3",
+    "pre_install": [
```
Comment (linsui) [inline fluent-terminal.json]:
```suggestion
    "depends": "sudo",
    "pre_install": [
```

**Pull request closed without merging**